src/Components/App/App.tsx

```diff
@@ -5,15 +5,22 @@ import BookCardContainer from '../BookCardContainer/BookCardContainer';
 import BookCard from '../BookCard/BookCard';
 
 // import { getBooksByCategory, getSeafood } from './apiCalls';
-import { getBookByHistory } from '../../apiCalls';
+import { getBookByCategory } from '../../apiCalls';
 // import { Route } from 'react-router-dom';
 import './App.css';
 import "./library.jpg"
 
 const App: React.FC = () => {
+  const [books, setBooks] = useState([])
+
+  const retrieveBooks = () => {
+    getBookByCategory('history')
+      .then((data: {works: []}) => setBooks(data.works))
+      .then(() => console.log(books))
```

Comment:
Just talked to my mentor, and he said this console.log probably wasn't working because the previous .then() isn't returning a promise... it's just setting the data. Because of that, when we opened the React dev tools, we were able to see the books successfully populate in state (even when the console log was empty)